typescript.js

```diff
@@ -4,7 +4,6 @@ module.exports = {
   },
   extends: [
     'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
Warum ist das Plugin hier entfernt worden? Weil `@typescript-eslint/recommended` davon extended?
`@typescript-eslint/recommended-requiring-type-checking` extended doch auch `@typescript-eslint/recommended`.

Reply:
Yep exactly. I will check the other packages too. I was only looking at the changes stated here: https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0